suite-native/qr-code/src/components/QRCodeCopyButton.tsx

```diff
@@ -5,25 +5,25 @@ import { useCopyToClipboard } from '@suite-native/helpers';
 
 type QRCodeCopyAndShareButtonProps = {
     data: string;
+    onCopy?: () => void;
     onCopyMessage: string;
-    onCopy: () => void;
 };
 
-export const QRCodeCopyAndShareButton = ({
+export const QRCodeCopyButton = ({
     data,
-    onCopyMessage,
     onCopy,
+    onCopyMessage,
 }: QRCodeCopyAndShareButtonProps) => {
     const copyToClipboard = useCopyToClipboard();
 
-    const handleCopyAndClose = async () => {
+    const handleCopy = async () => {
         await copyToClipboard(data, onCopyMessage);
-        onCopy();
+        onCopy?.();
     };
 
     return (
-        <Button size="large" onPress={handleCopyAndClose}>
-            Copy & Close
+        <Button size="large" onPress={handleCopy}>
```

Comment:
If copy callback is not present we should button probably

Reply:
Sorry, I don't understand your comment. Could you explain?

Reply:
Sorry did not understand that onCopy is just extra action that you can perform after thing is copied to clipboard, so then this makes sense.